app/models/project.rb

```diff
@@ -1,4 +1,10 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def average_age
+    self.contestants.average(:age)
```

Comment:
Good job with this AR, and putting it in your model!

Reply:
The only thing missing is your model tests. Whenever you create a new method in your model, you need to make sure it is tested in your model test as well.

Reply:
On that note, you also didn't turn in any feature tests at all. TDD is a requirement for both your project work and any assessments. Please get into the habit of writing your tests first, having them fail, and then writing your code to make those tests pass. Let us know if you are having trouble using that system.